### ISSUE TITLE
Can't find acf_add_local_field_group

### ISSUE THREAD

**Issue by permagnus** [elliotcondon/acf#412]
Hi,

I'm trying to use this plugin and more specifically the function acf_add_local_field_group according to the documentation at http://www.advancedcustomfields.com/resources/register-fields-via-php/ .

It seems like this function does not exist in the plugin or do I need to PRO version?

**Comment by elliotcondon** [comment]
Hi @permagnus 

Thanks for the issue.

The documentation you linked to is for ACF PRO ( I will a message today ) only.
The free ACF4 contains a function called `register_field_group`. You can use the Export tool to generate PHP code.

**Comment by atwellpub** [comment]
Will acf_add_local_field_group ever work with free?

**Comment by elliotcondon** [comment]
Hi @atwellpub

These are 2 changes made from v4 to v5 to better improve the plugin by future proofing functionality.
You can read more about changes here:
http://www.advancedcustomfields.com/resources/updates/upgrading-v4-v5/

Will acf_add_local_field_group work with free? No, this is a new function added in v5, there is a v4 siilar function register_field_group. Please export your field group from within the plugin and use the generated code.

**Comment by atwellpub** [comment]
I notice that current WordPress lite version is 4.4.3, but the article above mentions there is a 5 free version available. 

I'd be well to develop with it as a base, but where do I pick it up? Is 4.4.3 actually ACF5 free? I'm sorry for missing something that's probably obvious. 

When I find the free version of 5, acf_add_local_field_group will begin working? Or have I completely misunderstood and 4 will always be free and 5 will always be pro?

**Comment by elliotcondon** [comment]
ACF is currently available in 2 forms:
ACF - free - currently v4.4.3
ACF PRO - premium - currently 5.3.0

ACF PRO is a new version of ACF (v5), and will be released as a free update soon.

**Comment by atwellpub** [comment]
How about this, if we use register_field_group to register fields generated from ACF Pro instead of acf_add_local_field_group, will we lose any powers (repeaters/flexible content/etc.) ?

**Comment by elliotcondon** [comment]
Yes, if you swap out the function for ' register_field_group' all will work as expected!
